x-pack/test/detection_engine_api_integration/security_and_spaces/tests/create_rules.ts

```diff
@@ -120,6 +120,34 @@ export default ({ getService }: FtrProviderContext) => {
           expect(statusBody[body.id].current_status.status).to.eql('succeeded');
         });
 
+        it('should create a single rule with a deeply nested query and validate it failed', async () => {
+          const simpleRule = {
+            ...getRuleForSignalTesting(['auditbeat-*']),
+            query:
+              'not observer.egress:* and not observer.egress.zone:* and not observer.hostname:* and not observer.ingress:* and not observer.ingress.zone:* and not observer.ip:* and not observer.mac:* and not observer.name:* and not observer.product:* and not observer.serial_number:* and not observer.type:* and not observer.vendor:* and not observer.version:* and not agent.ephemeral_id:* and not agent.id:* and not agent.name:* and not agent.type:* and not agent.version:*',
+          };
+          const { body } = await supertest
+            .post(DETECTION_ENGINE_RULES_URL)
+            .set('kbn-xsrf', 'true')
+            .send(simpleRule)
+            .expect(200);
+
+          await waitForRuleSuccessOrStatus(supertest, body.id, 'failed');
+
+          const { body: statusBody } = await supertest
+            .post(DETECTION_ENGINE_RULES_STATUS_URL)
+            .set('kbn-xsrf', 'true')
+            .send({ ids: [body.id] })
+            .expect(200);
+
+          expect(statusBody[body.id].current_status.status).to.eql('failed');
+          expect(
+            statusBody[body.id].current_status.last_failure_message.includes(
```

Comment:
Something's going wrong here, `last_failure_message` doesn't seem to include the desired string but it's still returning true.  Using `expect.to.eql` causes this test to fail as I would expect given the error I'm seeing in the UI.

Reply:
Ah there's no assertion here - so `last_failure_message.includes` returns `false` but we need to also assert that it returns true.

Reply:
Fixed here: ba2a02c034f2684a48676dffdbf2f94a1fd3b6e7 Thanks for the eyes!